docs/v2/15_implementing-social-logins-to-webgl-builds.md

```diff
@@ -5,56 +5,56 @@ sidebar_label: Implementing Social Logins To  WebGL Builds
 ---
 
 
-# Implementing Social Logins To  WebGL Builds
+# Implementing Social Logins To WebGL Builds
 
 :::info
 
 How to import and use social logins with WebGL builds.
 
 :::
 
-## Why should we use social logins?
+## Why Should We Use Social Logins
 
-Some developers feel that having social logins would be a much better entrance system for users getting into the blockchain space. Some users tend to feel more comfortable having a wallet tied to an account they own. This helps to speed up onboarding and reduces friction when it comes to gaining new users. With a click of a button they can authorize an account and be on their way!
+Some developers feel that having social logins (e.g. Google, Facebook, Twitter, GitHub, Discord, etc.) would be a much better entrance system for users getting into the blockchain space. Some users tend to feel more comfortable having a wallet tied to a social account they already own. This helps to speed up onboarding and reduces friction when it comes to gaining new users. With a click of a button they can authorize an account and be on their way!
```

Comment:
We don't need to include "etc." when using "e.g." since it is by definition a list of examples

Reply:
Wow, I have always done it like that O_O ... learned something new. I have never looked at it as defining a list. My perception was using "e.g." to start off a bracketed section with "for example", and "etc" as extending like-for-like examples where the reader can fill in the blank

learned something new xD